packages/edit-site/src/components/dataviews/in-filter.js

```diff
@@ -31,49 +83,111 @@ export default ( { filter, view, onChangeView } ) => {
 			key={ filter.field }
 			trigger={
 				<Button variant="tertiary" size="compact" label={ filter.name }>
-					{ activeElement !== undefined
-						? sprintf(
-								/* translators: 1: Filter name. 2: filter value. e.g.: "Author is Admin". */
-								__( '%1$s is %2$s' ),
-								filter.name,
-								activeElement.label
-						  )
-						: filter.name }
+					<FilterSummary
+						activeElement={ activeElement }
+						filterInView={ filterInView }
+						filter={ filter }
+					/>
 					<Icon icon={ chevronDown } style={ { flexShrink: 0 } } />
 				</Button>
 			}
 		>
-			{ filter.elements.map( ( element ) => {
-				return (
-					<DropdownMenuCheckboxItem
-						key={ element.value }
-						value={ element.value }
-						checked={ activeElement?.value === element.value }
-						onSelect={ () =>
-							onChangeView( ( currentView ) => ( {
-								...currentView,
-								page: 1,
-								filters: [
-									...view.filters.filter(
-										( f ) => f.field !== filter.field
-									),
-									{
-										field: filter.field,
-										operator: OPERATOR_IN,
-										value:
-											activeElement?.value ===
-											element.value
-												? undefined
-												: element.value,
-									},
-								],
-							} ) )
+			<WithSeparators>
```

Comment:
This changes from:

```jsx
<DropdownMenu>
  <DropdownMenuCheckboxItem />
  <DropdownMenuCheckboxItem />
</DropdownMenu>
```

to:

```jsx
<DropdownMenu>
  <WithSeparators>
    <DropdownMenuGroup>
      <DropdownMenuCheckboxItem />
      <DropdownMenuCheckboxItem />
    </DropdownMenuGroup>
    <DropdownSubMenu>
      <DropdownMenuCheckboxItem />
      <DropdownMenuCheckboxItem />
    </DropdownSubMenu>
  </WithSeparators>
</DropdownMenu>
```

Essentially adds a new `DropdownSubmenu` tree. The rest are indentation changes that result of adding the `WithSeparators` component.